receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -515,6 +515,14 @@ metrics:
       value_type: int
     attributes: []
     enabled: true
+  # other jvm metrics, disabled by default
+  jvm.memory.heap.percentage:
```

Comment:
I think this should probably be `jvm.memory.heap.utilization`, based on the [naming guidelines](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/exporterhelper#persistent-queue).

Reply:
The link is invalid (it leads to the persistent queue), did you mean [this](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md#general-metric-semantic-conventions)?
I did change the name to `utilization`, but right in the guidelines stands `Utilization values are in the range [0, 1].`. Technically, this percentage is always in range `[0%,100%]` so it is the same as `[0, 1]`, but I'm not sure how strict that guideline is.

Reply:
Fixed.